### DISCUSSION TITLE
Tag und Nachtschaltung Display - Day and night switching Display

### DISCUSSION THREAD

**Discussion by Mercy-64** [Blackymas/NSPanel_HA_Blueprint#1825] [General]
Hi!
Ich habe bisher mit ioBroker das Lovelace UI für das NSPanel genutzt. Jetzt bin ich auf HA umgestiegen und habe das NSPanel mit ESPHome und dem Blueprint versehen. Soweit so gut! Was mir fehlt ist, dass das Panel in einer von mir festgelegten Zeitspanne (z.B. von 7:00 - 22:00 Uhr) normal hell ist bei Berührung und in den gedimmten Modus geht wenn eine Zeitspanne (30 Sekunden) abgelaufen ist. Ab 22:00 Uhr bis 7:00 Uhr soll das Panel komplett abdunkeln, aber noch bedienbar sein, und morgens um 7:00 Uhr wieder normal hell sein. Ist diese Funktion vorhanden oder muss dafür eine Automation erstellt werden? Ich bin da leider nicht fündig geworden.
Ich freue mich auf Anregungen :-)
Schöne Grüße
Mercy

**Comment by Mercy-64** [comment]
english text for question:
I used to use the Lovelace UI for the NSPanel with ioBroker. Now I've switched to HA and added ESPHome and the Blueprint to the NSPanel. So far so good! What I'm missing is that the panel is normally bright when I touch it during a period of time I specify (e.g. from 7:00 am to 10:00 pm) and goes into dimmed mode when a period of time (30 seconds) has elapsed. From 10 p.m. to 7 a.m. the panel should darken completely, but still be operable, and be normally bright again at 7 a.m. in the morning. Is this function available or does an automation have to be created for this? Unfortunately, I have not been able to find anything.
I look forward to your suggestions :-)
Best regards
Mercy

**Comment by f0b04707** [comment]
Have a look at discussion https://github.com/Blackymas/NSPanel_HA_Blueprint/discussions/1826 to setup the display as a light, then you can use a normal automation to turn on / off the screen.

**Reply by Mercy-64** [reply]
Nice project, but very complicated to set up the screen just to customize the screen saver. 
I know that many things are customizable, but if someone is not that much of a tinkerer, it gets really complicated. 
I would like to have a setting option in a config file where I can enter that the Display switches the brightness to aa at time xx and the brightness switches to down at time xy. 
This works perfectly in the Locelace UI of the panel and is easier to handle.
Otherwise a nice project, but unfortunately not usable for me in this form :-(
Regards Mercy

**Reply by edwardtfn** [reply]
> This works perfectly in the Locelace UI of the panel and is easier to handle.

It's nice that the solutions are different. 😉 
They have done a great job with Lovelace UI. It is a quite inspiring project.
We took a very different approach when implementing this one via Blueprint and standard Nextion commands. There are pros and cons in this approach, and this will certainly reflect on the pros and cons of each solution.

**Comment by edwardtfn** [comment]
It can be as complex as you want.

It could be as simple as having an automation to adjust "Brightness" at 7 a.m., then adjust to another level at 10 p.m.
It would be a very simple automation.

For dimming after 30s, it's just about adjust "Brightness - Dimming" to 30s. No need to an automation in this case, as this will be a fix value.

**Reply by Mercy-64** [reply]
I will try to set an automation for brightness - dimming the panel and will see how it works for me.
I have another question and i hope i am right here.
i am using two of the panels and one of them is in my normal network where HA Server is included and the other one is in my IoT Network. both networks has different IP scopes (normal network has 10.1.1.xx and IoT has 10.1.50.xx)
In my normal network the panel is working fine, i can see all icons, button etc. but my IoT network i am missing buttons and icons.
I'm using the same configuration on both panels. All devices can communicate between them.
Is there a problem with different networks?
Regards Mercy

**Reply by edwardtfn** [reply]
Different networks shouldn't be an issue as soon your HA is talking to your panel (ESPHome)
Why don't you report this as a bug in the issues area, so we can investigate it deeper?

**Reply by Mercy-64** [reply]
Because this was only a question and why should i open a new case when this is a configuration error?
But it looks like a bug, so i can open a new case, isn't it?

**Reply by edwardtfn** [reply]
I was talking about the different network thing. It should work. If it isn't working, let's investigate it as if it is a bug. 😉

**Reply by Mercy-64** [reply]
Should i open a case for this?

**Reply by edwardtfn** [reply]
I think so.

**Reply by Mercy-64** [reply]
Hi!
I created an automation to setup the dim mode and screensaver mode. I hope it will working for me.
Thanks for this helpful tips.
Regards Mercy
